operations/helm/charts/mimir-distributed/values.yaml

```
@@ -4075,9 +4075,9 @@ tokengenJob:
   annotations: {}
   initContainers: []
   # -- Controls whether to store the generated admin token in a Kubernetes secret
-  storeTokenInSecret: true
+  storeTokenInSecret: false
   # -- Name of the secret to store the admin token. If not specified, defaults to "<release-name>-admin-token"
```

the comment for the default looks redundant now


```suggestion
  # -- Name of the secret to store the admin token.
```